package.json

```diff
@@ -11,8 +11,20 @@
     "packages/*"
   ],
   "devDependencies": {
+    "all-contributors-cli": "^6.8.1",
     "lerna": "^3.17.0",
-    "react": "16.9.0",
-    "all-contributors-cli": "^6.8.1"
+    "lerna-changelog": "^0.8.2",
+    "react": "16.9.0"
+  },
+  "changelog": {
+    "repo": "lerna/lerna-changelog",
+    "labels": {
+      "breaking": ":boom: Breaking Change",
+      "enhancement": ":rocket: Enhancement",
+      "bug": ":bug: Bug Fix",
+      "documentation": ":memo: Documentation",
+      "internal": ":house: Internal"
+    },
+    "cacheDir": ".changelog"
```

Comment:
Can you add `ignoreComitter` too, to ignore the PR's that are opened by *all-contributors*, since those are just PR's from bot.